etc/fetchmail.profile

```diff
@@ -0,0 +1,29 @@
+# Firejail profile for fetchmail
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/fetchmail.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+
```

Comment:
Could `blacklist /tmp/.X11-unix` be a useful addition?

Reply:
It would be required to use net none as well afaik to actually be effective.

Reply:
You're right! It only effective in conjunction with `net *` or that `-nolisten local` xserver option.

Reply:
@SpotComms I close these comments